hs_composite_resource/tests/test_composite_resource.py

```diff
@@ -3444,6 +3444,55 @@ def test_version_resource_immunity_unpublished(self):
         )
         self.assertFalse(ori_res.raccess.immutable)
 
+    def test_version_resource_published_deleted_user(self):
```

Comment:
Should there be a test for versioning an unpublished resource with a deleted creator?